contracts/modules/SSVOperatorsWhitelist.sol

```diff
@@ -0,0 +1,113 @@
+
+        emit OperatorWhitelistUpdated(operatorId, whitelistAddress);
+    }
+
+    function setOperatorMultipleWhitelists(
+        uint64[] calldata operatorIds,
+        address[] calldata whitelistAddresses
+    ) external override {
+        OperatorLib.updateMultipleWhitelists(whitelistAddresses, operatorIds, true, SSVStorage.load());
+        emit OperatorMultipleWhitelistUpdated(operatorIds, whitelistAddresses);
+    }
+
+    function removeOperatorMultipleWhitelists(
+        uint64[] calldata operatorIds,
+        address[] calldata whitelistAddresses
+    ) external override {
+        OperatorLib.updateMultipleWhitelists(whitelistAddresses, operatorIds, false, SSVStorage.load());
+        emit OperatorMultipleWhitelistRemoved(operatorIds, whitelistAddresses);
+    }
+
+    function setOperatorsWhitelistingContract(
+        uint64[] calldata operatorIds,
+        ISSVWhitelistingContract whitelistingContract
+    ) external {
+        // Reverts also when whitelistingContract == address(0)
+        if (!OperatorLib.isWhitelistingContract(address(whitelistingContract)))
+            revert InvalidWhitelistingContract(address(whitelistingContract));
+
+        uint256 operatorsLength = OperatorLib.checkOperatorsLength(operatorIds);
+
+        StorageData storage s = SSVStorage.load();
+        Operator storage operator;
+
+        for (uint256 i; i < operatorsLength; ++i) {
+            uint64 operatorId = operatorIds[i];
+
+            operator = s.operators[operatorId];
+            operator.checkOwner();
+
+            address currentWhitelisted = s.operatorsWhitelist[operatorId];
+
+            // operator already whitelisted?
+            // if EOA or generic contract, move it to SSV whitelisting module
+            if (!OperatorLib.isWhitelistingContract(currentWhitelisted)) {
+                (uint256 blockIndex, uint256 bitPosition) = OperatorLib.getBitmapIndexes(operatorId);
+                s.addressWhitelistedForOperators[currentWhitelisted][blockIndex] |= (1 << bitPosition);
+            }
+
+            s.operatorsWhitelist[operatorId] = address(whitelistingContract);
+        }
+
+        emit OperatorWhitelistingContractUpdated(operatorIds, address(whitelistingContract));
+    }
+
+    function removeOperatorsWhitelistingContract(uint64[] calldata operatorIds) external {
+        uint256 operatorsLength = OperatorLib.checkOperatorsLength(operatorIds);
+
+        StorageData storage s = SSVStorage.load();
+        Operator storage operator;
+
+        for (uint256 i; i < operatorsLength; ++i) {
+            uint64 operatorId = operatorIds[i];
+            operator = s.operators[operatorId];
+
+            operator.checkOwner();
+
+            s.operatorsWhitelist[operatorId] = address(0);
+        }
+
+        emit OperatorWhitelistingContractUpdated(operatorIds, address(0));
+    }
+
+    function setOperatorsPrivateUnchecked(uint64[] calldata operatorIds) external override {
```

Comment:
chatted with @arielssv on this name, i dont like it lol, he likes it, just leaving a comment that i think its very confusing, i imagine that private has a checkmark and we are unchecking it ie making it public

Reply:
@andrew-blox of course, all comments are welcomed and I'm all in to get a better name. You know the 
 meaning of the `unchecked` suffix: don't check if the operator has any whitelisted address.
`setOperatorsPrivateUnsafe` for me sounds too aggressive.
maybe just `setOperatorsPrivate`? The doc of the function says:
```
/// @notice Set the list of operators as private without checking for any whitelisting address
/// @notice The operators are considered private when registering validators
```
    cc @arielssv